src/components/form/widgets/input-field.e2e.ts

```diff
@@ -0,0 +1,78 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('form input-field', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    const schema = {
+        title: 'Registration form',
+        type: 'object',
+        properties: {
+            name: {
+                type: 'string',
+                title: 'Name',
+                default: '',
+                description: 'Enter your heroic name',
+            },
+        },
+    };
+    const props = { schema };
+
+    await page.setContent('<div></div>');
+    await page.$eval('div', (elm: any, { schema }) => {
+        const form = document.createElement('limel-form');
+        form.schema = schema;
+        elm.appendChild(form);
+    }, props);
+
+    await page.waitForChanges();
+    const form = await page.find('limel-form');
+    expect(form).toHaveClass('hydrated');
+
+    const limelInput = await page.find('limel-form >>> limel-input-field');
+    expect(limelInput).toHaveClass('hydrated');
+
+    await page.waitForChanges();
+    await page.waitForChanges();
+    await page.waitForChanges();
+    const inp = await form.find('limel-input-field >>> .mdc-text-field__input');
+    console.log('input', inp, '\n\n-----\n\n-----\n\n-----\n\n');
+    console.log('\n\n-----\n\n-----\n\n-----\n\n');
+
+    // expect(form).toEqualHtml('<limel-input-field></limel-input-field>');
+    // expect(limelInput).toEqualHtml('<limel-input-field></limel-input-field>');
```

Comment:
If I uncomment this line, I get an error that tells me the html of `limelInput` looks like this:

```html
<limel-input-field class="hydrated" format-number="" helper-text="Enter your heroic name" label="Name" step="any" type="text" value="">
  <mock:shadow-root>
    <label class="mdc-ripple-upgraded mdc-text-field">
      <input class="mdc-text-field__input" type="text">
      <span class="mdc-floating-label">
        Name
      </span>
      <div class="mdc-line-ripple"></div>
    </label>
    <div class="mdc-text-field-helper-line">
      <p class="mdc-text-field-helper-text">
        Enter your heroic name
      </p>
    </div>
    <div class="autocomplete-list-container"></div>
  </mock:shadow-root>
</limel-input-field>
```

Despite that, line 37 (`const inp = await form.find('limel-input-field >>> .mdc-text-field__input');`) finds nothing. I've tried several different versions, but I'm getting nowhere at the moment.